docs/reference/backends.md

```diff
@@ -258,6 +258,7 @@ Current implemented algorithms:
 - `roundRobin`: backend is chosen by the round robin algorithm, starting with a random selected backend to spread across all backends from the beginning
 - `random`: backend is chosen at random
 - `consistentHash`: backend is chosen by a consistent hashing algorithm with the client X-Forwarded-For header with remote IP as the fallback as input to the hash function
+- `powerOfRandomNChoices`: backend is chosen by powerOfRandomNChoices algorithm with randomly select N backends and choose less active one.
```

Comment:
Shouldn't this description, this https://github.com/zalando/skipper/pull/1548/files#diff-861ba78658f4e6d2a609b032484d23b1R22
and this https://github.com/zalando/skipper/pull/1548/files#diff-8ef30ff734de1bf682a3b0d530f9f663R34 all match?

Reply:
Also it might make sense to add a link to algorithm paper